pkg/volume/testing/testing.go

```diff
@@ -1229,15 +1229,14 @@ func FindEmptyDirectoryUsageOnTmpfs() (*resource.Quantity, error) {
 		return nil, err
 	}
 	defer os.RemoveAll(tmpDir)
-	out, err := exec.New().Command("nice", "-n", "19", "du", "-x", "-s", "-B", "1", tmpDir).CombinedOutput()
+	out, err := exec.New().Command("nice", "-n", "19", "du", "-x", "-s", "-k", tmpDir).CombinedOutput()
```

Comment:
what this change for?

Reply:
Sounds like I need to improve the commit message.

While looking at issue #95172 I grep'ed the code for other calls to `du` and found this place.

It fixes du command to run on Alpine Linux which uses busybox instead of GNU coreutils. (#95172)

The `-B 1` argument is a GNU extension, not defined in POSIX du[1] and does not work with
busybox du used by Alpine Linux. Fix this by using `-k`, which is defined in POSIX[1] and portable and append "Ki" for the ParseQuantity call so du's output is properly parsed.

`resource.ParseQuantity()` will set the format to `BinarySI` when there
is a "Ki" suffix so we can remove the explicit set of the format.

[1]: https://pubs.opengroup.org/onlinepubs/9699919799/utilities/du.html

Maybe I should do this in separate PR? I'm not sure where/when `pkg/volume/testing`  is used, but I will definitively fail on Alpine Linux unless you install coreutils. With this change it will be portable to anything POSIX.

Or maybe we should let it call the go implementation instead to get rid of the external `du` dependency?

Reply:
I think go implementation is better. You can either change here or make a separate PR.

Reply:
I had a look at this, but this is only used by the unit tests. I think it makes more sense to call the external `du` to verify that the go implementation in this PR corresponds with what is expected from `du` instead of comparing two identical calls to the same function, so I don't think we should use the go implementation here.

I think my commit is slightly better than what it currently does, but I'm ok to remove it and require GNU coreutils for running the unit tests.

Reply:
actually, lets deal with this in separate PR.